std_msgs/Header header
float64 cam_std_alt               # cam_std_alt reading
float64 cam_std_alt_var           # cam_std_alt_var reading


